Controller/TutorialController.php

```diff
@@ -41,6 +42,19 @@ public function labelsAction()
     }
 
     /**
+     * @Route("/tutorial/button-icons")
+     * @Template()
+     */
+    public function buttonIconsAction()
+    {
+        $form = $this->createForm(new ButtonIconType);
+
+        return array(
```

Comment:
in the buildForm method, you use new array syntax. I would choose one of both types. You're still inconsistent now (just not in the same file)